spec_driven_model/models/spec_import.py

```diff
@@ -196,6 +178,18 @@ def _prepare_import_dict(self, vals, model=None):
         """
         if model is None:
             model = self
+
+        defaults = model.with_context(parent_dict=parent_dict).default_get(
+            [
+                f
+                for f, v in model._fields.items()
+                if v.type not in ["binary", "integer", "float", "monetary"]
```

Comment:
Eu estou tendo um problema com a integração com o módulo l10n_br_delivery_nfe. Lá é adicionado um valor default para o campo nfe40_modFrete, e este valor está sobrescrevendo o valor importado do xml.
Acho que adicionar o tipo select nesse if resolveria o problema. Existe algum motivo para ele não estar aqui?

Reply:
@hirokibastos o `and v.name not in vals.keys()` da linha depois era exactamente para evitar de sobre-escrever algum campo vindo do XML. Vc sabe porque essa linha não solta o efeito no seu caso? o campo `nfe40_modFrete` não esta no vals neste caso?

Reply:
@hirokibastos talvez deveriamos mudar esse default_get pro fim do _prepare_import_dict, não é?

Reply:
@hirokibastos seu problema vem desse compute então https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_delivery_nfe/models/document.py#L12 ? Se vc tirar o compute funciona?
Não seria o caso de adicionar um inverse="inverse_nfe40_modFrete" no modulo l10n_br_delivery_nfe então?  Olha por examplo o metodo _inverse_nfe40_tpNF aqui https://github.com/OCA/l10n-brazil/blob/12.0/l10n_br_nfe/models/document.py#L273 não seria o mesmo tipo de situação?

Reply:
Esse compute provavelmente também irá ser problematico. Mas o problema atual acontece antes dele ser rodado. O dicionário que retorna da chamada do build_attrs dentro do método build já está com o valor incorreto.
https://github.com/OCA/l10n-brazil/blob/fcc70ed3f5fc91088a4a990cdd18c848391a7c8f/spec_driven_model/models/spec_import.py#L46

Passar o default_get para o final pode ser uma possível solução. Vou rodar uns testes e ver se encontro o motivo do modFrete não estar no vals.

Reply:
Ola @hirokibastos eu mudei o default_get pro fim do _prepare_import_dict de qualquer forma e fiz um push -f agora (tb passo o record_dict no default_get que pode ajudar quem sobreescreve o default_get).

Me informa como vc ta indo com esse assunto porque seria interessante a gente se assegurar que isso funciona...

Reply:
@hirokibastos , se vc tiver algum XML para importar com algum pedaçõ de codigo/Odoo shell para mim testar na 14.0, pode me passar que eu posso ver tb... Ou ate na 12.0...